### ISSUE TITLE
Thermal Series dusts

### ISSUE THREAD

**Issue by toxicjayce** [FTBTeam/FTB-Modpack-Issues#2341]
### Modpack

FTB Presents Direwolf20 1.19

### Modpack version

1.0.1

### Describe the bug

Thermal Series dusts from the pulverizer are unable to be used in crafting recipes for alloy metals and other crafting recipes.

Specific issues I ran into were, Thermals copper dust being unable to be used in the crafting of bronze, Thermals iron dust being unable to be used in the crafting of invar, Thermals nickel dust doesn't have any crafting or smelting recipes this also includes silver, lead, tin, and possibly others.

When using JEIs uses and recipes hotkey either nothing shows or only a few or a single recipe are available to be used with said item.

TLDR; Much of Thermal Series crafting recipes with metal processing are not working or are only somewhat working.

![Screenshot 2022-12-16 041605](https://user-images.githubusercontent.com/120718533/208077025-1fc38f40-e0ed-4f67-b282-a8882e00e9cd.png)

![Screenshot 2022-12-16 041630](https://user-images.githubusercontent.com/120718533/208077044-8b01a8ef-2e02-4721-b588-f1ee17e3512a.png)

### Steps to reproduce

1. Pulverize silver, lead, tin, nickel, iron
2. Try to use dust in crafting recipes (or smelting for nickel and possibly others)
3. Voilà

### Expected behaviour

Able to be used in crafting and/or smelting.

**Comment by ukmcplyr** [comment]
This (and FTBTeam/FTB-Modpack-Issues#2337) is mostly covered by FTBTeam/FTB-Modpack-Issues#2286 and FTBTeam/FTB-Modpack-Issues#2288.
